### ISSUE TITLE
Distance and Time for Last Lightning Strike disappear after sensor/hub replacement

### ISSUE THREAD

**Issue by mmccool** [briis/weatherflow_forecast#167]
**Describe the bug**
Installed a new Tempest Weather Station, as the battery on the old one was failing, and after generating a new set of entities (I changed the station name, probably shouldn't have) there are no entities for distance_last_lightning_strike and time_of_last_lightning_strike.  Are these no longer in the API?  Are they no longer supported on the new hardware?  They were showing up on my old device.   I also got a new hub.

**To Reproduce**
1. Buy a new Tempest Weather station
2. Follow the instructions at https://help.tempest.earth/hc/en-us/articles/360051181713-Managing-Your-Tempest-Account for "Merging Stations or Replacing Hubs"
3. Verify that everything works via the app.
4. Reconfigure and restart integration.

**Expected behavior**
I would have expected the same set of entities to show up, but the two noted are now missing.

**Screenshots**

**Additional context**
I went through all the entities to see if they were just under a different name now but could not find them.  I checked the Tempest docs and it does not seem like this feature has been deprecated.

**Comment by mmccool** [comment]
<img width="1131" alt="Screenshot 2024-09-19 174925" src="https://github.com/user-attachments/assets/c1f4b976-d1f8-40a1-b913-c2808956d133">

**Comment by mmccool** [comment]
Hmmm... I just realized a brand new device would not HAVE any "last" strikes.  So maybe after I get one these entities will appear.  It would be nicer though if they were available but with some very large default value for the purposes of setting up UIs.   But it's probably just that these entities are not available YET.

**Comment by briis** [comment]
Yes. Unfortunately Lightning Strike sensors do not contain any values before they have measured anything the first time. This is per design of WeatherFlow so not a lot I can do about it.

**Comment by briis** [comment]
This will closed now, as this is per design of WeatherFlow
